packages/renderless/src/input/index.ts

```
@@ -147,6 +156,17 @@ export const calcTextareaHeight =
     const singleRowHeight = hiddenTextarea.scrollHeight - paddingSize
 
     if (minRows !== null) {
+      // 新增逻辑：单行时直接用 单行input高度
+      const inputSizeMf = props.inputSizeMf || props.size || 'medium'
+      if (minRows === 1) {
+        const minHeight = getMinHeightByInputSizeMfAndRows(inputSizeMf, 1)
+        textareaStyle.minHeight = minHeight
+        textareaStyle.height = minHeight
+        hiddenTextarea.parentNode && hiddenTextarea.parentNode.removeChild(hiddenTextarea)
+        hiddenTextarea = null
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)